### PR TITLE
Open in window

### PR DESCRIPTION
**Pull request by MichaelBailly**
ezgif-3-17134bc622d9

### CODE REVIEW

File: src/lib/directive.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const _ = require('lodash');
 
   angular.module('esn.mailto-handler').directive('opInboxCompose', opInboxCompose);
 
-  function opInboxCompose($parse, $window) {
+  function opInboxCompose($parse, $window, MAILTOHANDLER_CONSTANTS) {
```
Comment (rezk2ll) [inline src/lib/directive.js:8]: we need to `require('./constant.js')`

Comment (MichaelBailly) [reply src/lib/directive.js:8]: I did it in the component, isn't it OK ?

**Review by huy-ta — COMMENTED** [review]
Great enhancement. I wonder if my added pages will be nicely responsive :trollface:.

How does this work on mobile devices though?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
...
+```html
+<a op-inbox-compose op-inbox-compose-users='[{email: "jdoe@acme.com"}]' >Send mail</a>
+```
+
+* { prefferedEmail }
+
+Example:
+
+```html
+<a op-inbox-compose op-inbox-compose-users='[{prefferedEmail: "jdoe@acme.com"}]' >Send mail</a>
```
[the remark this answers is not in the document]
Comment (MichaelBailly) [reply README.md]: Thanks for your review ! The correct spelling is preferredEmail :-)

**Review by rezk2ll — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Review by huy-ta — APPROVED** [review]

**Pull request merged**